### PR TITLE
feat(ui): DOMA-4634 Added carousel component

### PR DESCRIPTION
**Pull request by SavelevMatthew**

### CODE REVIEW

File: packages/ui/src/components/Carousel/carousel.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+export const Carousel: React.FC<CarouselProps> = (props) => {
+    const {
+        children,
+        autoplaySpeed = DEFAULT_AUTOPLAY_SPEED,
+        ...rest
+    } = props
+
+    return (
+        <div className='.wowow'>
```
Comment (sitozzz) [inline packages/ui/src/components/Carousel/carousel.tsx:33]: Strange class name

Comment (SavelevMatthew) [reply packages/ui/src/components/Carousel/carousel.tsx:33]: Ooopsy, left from testing 😄

Comment (SavelevMatthew) [reply packages/ui/src/components/Carousel/carousel.tsx]: Fixed

File: packages/ui/src/components/Carousel/arrow.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+export const CarouselArrow: React.FC<React.HtmlHTMLAttributes<HTMLDivElement>> = (props) => {
+    return (
+        <div {...props}>
+            {/*TODO (DOMA-4666): Move to icons pack*/}
+            <svg width='9' height='14' fill='none' xmlns='http://www.w3.org/2000/svg'>
+                <path d='M1 2.374 5.414 7 1 11.626 2.293 13 8 7 2.293 1 1 2.374Z' fill='currentColor' stroke='currentColor' strokeWidth='.7'/>
```
Comment (toplenboren) [inline packages/ui/src/components/Carousel/arrow.tsx:8]: best to move to const

File: packages/ui/src/components/Carousel/carousel.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    return (
+        <DefaultCarousel
+            {...rest}
+            {...DEFAULT_ARROW_PROPS}
+            autoplaySpeed={autoplaySpeed}
+            prefixCls={CAROUSEL_CLASS_PREFIX}
+        >
+            {React.Children.map(children, (child, index) => {
+                return (
+                    <div key={index}>
```
Comment (toplenboren) [inline packages/ui/src/components/Carousel/carousel.tsx:41]: best to use section here? maybe we use UI kit in website builder or something.. looks like a small change that would make seo indexing a little bit better

**Review by toplenboren — APPROVED** [review]

**Review by Alllex202 — APPROVED** [review]
add result screenshot pls

File: packages/ui/src/components/Carousel/arrow.tsx (hunk 2 above)
Comment (SavelevMatthew) [reply packages/ui/src/components/Carousel/arrow.tsx:8]: I think it's not that big deal right now, since it's going to move to icon pack later

**Review by AleX83Xpert — APPROVED** [review]

File: packages/ui/src/stories/Carousel.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import { ComponentStory, ComponentMeta } from '@storybook/react'
+import { styled } from '@storybook/theming'
+import { get } from 'lodash'
+import { Carousel as Component } from '@open-condo/ui/src'
+import { colors } from '@open-condo/ui/src/colors'
+
+const AVAILABLE_COLORS = [
+    colors.green['3'],
```
Comment (sitozzz) [inline packages/ui/src/stories/Carousel.stories.tsx:9]: String numbers as keys looks little bit strange here

File: packages/ui/src/stories/Carousel.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+    colors.blue['3'],
+    colors.red['3'],
+    colors.teal['3'],
+    colors.cyan['3'],
+]
+
+const Slide = styled.h1<{ index: number }>`
+  background: ${(props => AVAILABLE_COLORS[props.index % AVAILABLE_COLORS.length])};
+  border-radius: 8px;
+  height: 320px;
```
Comment (sitozzz) [inline packages/ui/src/stories/Carousel.stories.tsx:19]: May this become editable from props?

File: packages/ui/src/stories/Carousel.stories.tsx [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+    colors.green['3'],
+    colors.blue['3'],
+    colors.red['3'],
+    colors.teal['3'],
+    colors.cyan['3'],
+]
+
+const Slide = styled.h1<{ index: number }>`
+  background: ${(props => AVAILABLE_COLORS[props.index % AVAILABLE_COLORS.length])};
+  border-radius: 8px;
```
Comment (sitozzz) [inline packages/ui/src/stories/Carousel.stories.tsx:18]: This is a default border radius for all of the components. Maybe move this value to constant styles?

File: packages/ui/src/stories/Carousel.stories.tsx (hunk 5 above)
Comment (SavelevMatthew) [reply packages/ui/src/stories/Carousel.stories.tsx:19]: It's a story, not component itself :)
Slide is just a random content of a carousel

Comment (sitozzz) [reply packages/ui/src/stories/Carousel.stories.tsx:19]: You right, my bad)

File: packages/ui/src/stories/Carousel.stories.tsx (hunk 6 above)
Comment (sitozzz) [reply packages/ui/src/stories/Carousel.stories.tsx:18]: Same as above I suppose

**Pull request merged**